src/Connector.sol

```diff
@@ -139,15 +150,33 @@ contract CentrifugeConnector is Test {
         memberlist.updateMember(user, validUntil);
     }
 
-    function transferTo(
+    function handleTransfer(
         uint64 poolId,
         bytes16 trancheId,
         address user,
         uint256 amount
     ) public onlyRouter {
         RestrictedTokenLike token = RestrictedTokenLike(tranches[poolId][trancheId].token);
+        require(address(token) != address(0), "CentrifugeConnector/unknown-token");
```

Comment:
Why `require(address(token) != address(0)` ?

Reply:
You can't transfer tokens if the token contract not exist yet on this domain. In solidity, there's no options/nullable variables, rather everything is defaulted to it's zero-like value. So address(0) is what a map to an address will be set to by default, indicating "empty".

Reply:
oh I see, it's like ensuring the address of the token `!= null`. Thanks!